start_ocean.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+export OCEAN_VERSION=latest
+# Must be set to true for the first run, change it to "false" to avoid migrating the smart contracts on each run.
+export DEPLOY_CONTRACTS="true"
+# Ganache specific option, these two options have no effect when not running ganache-cli
+export GANACHE_DATABASE_PATH="."
```

Comment:
This folder reference to the $PWD where the user run the script from, not the script's folder. To get the script's folder `DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )"` can be used.

Reply:
My intention was the CWD or $PWD.